CFC_DataCollector/recommender/traffic.py

```diff
@@ -0,0 +1,40 @@
+## A wrapper for the 511 traffic api
+import xml.etree.ElementTree as ET
+import urllib2, datetime
+
+def get_orig(orig_city):
+	url = "http://services.my511.org/traffic/getoriginlist.aspx?token=76495c63-cb01-4aac-af28-f717ffee11c8"
```

Comment:
Sounds good,
we dont need it

On Wed, May 6, 2015 at 3:13 AM, shankari <notifications@github.com> wrote:

> In CFC_DataCollector/recommender/traffic.py
> <https://github.com/e-mission/e-mission-server/pull/111#discussion_r29747300>
> :
>
> > @@ -0,0 +1,40 @@
> > +## A wrapper for the 511 traffic api
> > +import xml.etree.ElementTree as ET
> > +import urllib2, datetime
> > +
> > +def get_orig(orig_city):
> > +	url = "http://services.my511.org/traffic/getoriginlist.aspx?token=76495c63-cb01-4aac-af28-f717ffee11c8"
>
> The token shouldn't be hardcoded and checked in.
> Other projects will need other tokens, and we don't want to share our
> token and leave it open to abuse.
>
> @joshzarrabi <https://github.com/joshzarrabi>, I am removing this file
> for now since it is not being used, please fix later.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/e-mission/e-mission-server/pull/111/files#r29747300>.
>